package.json

```diff
@@ -3,8 +3,15 @@
   "version": "1.0.0",
   "description": "Weather app with data from open weather api",
   "main": "index.js",
+  "engines": {
+    "npm": "6.x",
+    "node": "14.x"
+  },
```

Comment:
Is there a specific reason why are define the engine object here? Leaving this way, any person with prior versions of Node 14 will not be able to use your app.

Reply:
Hi Sergio, 
Thanks for the approval!
I used this as the heroku docs specified this definition is required before deployment. 
I opted for firebase hosting so I will remove this definition before I merge this branch.